src/imaginary/test/test_actions.py

```diff
@@ -3,10 +3,25 @@
 Unit tests for Imaginary actions.
 """
 
+from zope.interface import (
```

Comment:
The import formatting style here is different, was that intentional?

Reply:
Oops, not really I guess, just habit from other projects.  I suppose the modern thing to do would be something involving black.  Maybe I'll take that on separately sometime.